01-basics.md

```diff
@@ -6,13 +6,13 @@ minutes: 5
 ---
 > ## Learning Objectives {.objectives}
 >
-> *   Understand the benefits of an automated version control system
-> *   Understand the basics of how Git works
+> *   Understand the benefits of an automated version control system.
+> *   Understand the basics of how Git works.
 
 We'll start by exploring how version control can be used
 to keep track of what one person did and when.
 Even if you aren't collaborating with other people,
-version control is much better for this than this:
+automated version control is much better than this:
```

Comment:
You may wish to replace 'this' with a concrete thought (undocumented changes, unreproducible history, etc.)